backend/src/controllers/popuplog.controller.js

```
@@ -1,27 +1,14 @@
 const popupService = require("../service/popuplog.service.js");
 const { internalServerError } = require("../utils/errors.helper.js");
+const { internalServerError } = require("../utils/errors.js");
```

Biome

[error] Shouldn't redeclare 'internalServerError'. Consider to delete it or rename it.

'internalServerError' is defined here:


(lint/suspicious/noRedeclare)